core/app/(user)/signup/page.tsx

```diff
@@ -12,7 +12,7 @@ export default async function Page() {
 		return (
 			<div>
 				You are not allowed to signup for an account, or the link that you clicked is
-				invalid expired. Please try again
+				invalid or expired. Please try again
```

Comment:
should they really try again?

Reply:
No, I guess not. I wanted to capture something along the lines of "click here to send another email"/"ask the person who invited you to try again", but that's rather complex

Reply:
ok that makes sense! sorry if that sounded critical, I just meant that I didn't actually remember what they're supposed to do at that point. something to revisit as we build out the permissions!